torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -975,6 +1023,10 @@ def __init__(
         self._materialize_module(module, param_init_fn, device_from_device_id)
         self._move_module_to_device(module, ignored_params, device_from_device_id)
         self.compute_device = self._get_compute_device(module, ignored_params, device_from_device_id)
+        self._max_inflight_all_gather_size = (
```

Comment:
@lessw2020 This is the internal knob to tune for this size-based limit.

Reply:
got it, thanks!